guides/index.md

```diff
@@ -6,6 +6,7 @@ title: skinship | Guides
 
 # Guides
 
+-   [Centring Accuracy on the Playfield](./centering_accuracy)
```

Comment:
why is the file named "centering" eventho it title and co use "centring" 🤔

Reply:
fixed